src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -612,15 +597,17 @@ TEST_CASE("request_rate_distribution")
 }
 
 /// Check that the request distribution is correct
-/// for the case where the measurement window is tiny
-/// and thus the code will loop through the schedule
-/// many times
+/// for the case where the measurement window is tiny.
+/// The code Used to loop through the schedule many times, and
+/// this exposed a bug. The code no longer needs to loop through
+/// the schedule, but the test is still valid
```

Comment:
Is it necessary to mention a past bug in the comment of this test? I think it should just describe what the test is testing, agnostic of past bugs.